tests/performance/loadtest/loadtest_suite_test.go

```diff
@@ -76,27 +78,21 @@ func TestEdgecoreK8sDeployment(t *testing.T) {
 		str2 := strconv.FormatInt(int64(nodePort), 10)
 		cloudHub = "wss://" + cloudCoreHostIP + ":" + str2
 		//Deploye edgecore as a k8s resource to cluster-2
-		podlist = HandleEdgeDeployment(cloudHub, ctx.Cfg.ApiServer2+DeploymentHandler, ctx.Cfg.ApiServer2+NodeHandler,
+		podlist = HandleEdgeDeployment(cloudHub, ctx.Cfg.ApiServer2+DeploymentHandler, ctx.Cfg.ApiServer+NodeHandler,
```

Comment:
here podlist is of no use, you can remove it..

Reply:
the `podlist` will used in the `AfterSuite`